README.rst

```diff
@@ -0,0 +1,262 @@
+##################################################
+MVS - Multi-Vector Simulator of the E-Land toolbox
+##################################################
+
+|badge_docs| |badge_CI| |badge_coverage|
+
+Depreciated: |badge_travis|
+
+Rights: `Reiner Lemoine Institut (Berlin) <https://reiner-lemoine-institut.de/>`__
+
+The multi-vector simulator (MVS) allows the evaluation of local sector-coupled energy systems that include the energy carriers electricity, heat and/or gas. The MVS has three main features:
```

Comment:
Please replace all occurrences with small letters with the capitalized ones!
 
```suggestion
The Multi-Vector Simulator (MVS) allows the evaluation of local sector-coupled energy systems that include the energy carriers electricity, heat and/or gas. The MVS has three main features:
```

Reply:
Done